Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -14,7 +14,7 @@ $telemetryCodes =
   "PREREQ_InvalidFilePath" = "PREREQ_InvalidFilePath";
   "PREREQ_StorageAccountNotFound" = "PREREQ_StorageAccountNotFound";
   "PREREQ_NoVMResources" = "PREREQ_NoVMResources";
-  "PREREQ_UnsupportedAzurePSVerion" = "PREREQ_UnsupportedAzurePSVerion";
```

Comment:
Till now our telemetry results were not coming at all so i thought changing now is not a problem